docs/interactions/connecting/index.md

```diff
@@ -1,6 +1,6 @@
 JanusGraph can be queried from all languages for which a TinkerPop
 driver exists. Drivers allow sending of Gremlin traversals to a Gremlin
-Server like the [JanusGraph Server](../basics/server.md). A list of TinkerPop
+Server like the [JanusGraph Server](../../operations/server.md). A list of TinkerPop
```

Comment:
Shall we put the "JanusGraph Server" chapter close to/under the "Connecting to JanusGraph" category?

Reply:
Florian mention something similar, see https://github.com/JanusGraph/janusgraph/pull/2348#discussion_r553969789 .

I would like to do it in an follow up PR. From my perspective running the Janusgraph Server should be under operations, how to configure and operate it.